README.md

```diff
@@ -108,7 +108,7 @@ async def main() -> None:
 
         # Extract data from the page.
         data = {
-            'url': context.request.url,
+            'url': str(context.request.url),
```

Comment:
so was this a string before #343? if so, it feels quite breaking. also i am honestly not sure if its a good idea to have something else than a string URL in the context